README.md

```diff
@@ -1,11 +1,11 @@
-# Artefacts-Utils Rosbag GPL package
+# Artefacts Toolkit Rosbag GPL package
 
-Designed to be used with `artefacts-utils` this package contains code available under the GPL license.
+Designed to be used with `artefacts-toolkit` this package contains code available under the GPL license.
```

Comment:
Does it work standalone too, as a library on its own? Wondering whether there is transitivity of licence risk.

Reply:
It does yes. and the README notes how to use standalone. 

Perhaps we should just not include in `artefacts-toolkit-rosbag` and `artefacts-toolkit` (so no umbrella wrapping). I believe that would make it "unmodified" when used, and thus not affecting licensing of the cli (if we include it in that) (?)